plugins/opa/examples/cert-expiration/deployments.yaml

```diff
@@ -0,0 +1,10 @@
+clusters: null
+output: null
+parameters:
+  days: 60
+runEnvironments: null
+targets:
+  - apiGroups:
+      - ""
+    kinds:
+      - Secret
```

Comment:
We should probably change the name of this file, either `certs` or `secrets` or something would work.

Reply:
Changed to certs.yaml